septa-fare-calculator/fares.json

```diff
@@ -1,155 +0,0 @@
-{
-    "info": {
-        "anytime": "Valid anytime",
-        "weekday": "Valid Monday through Friday, 4:00 a.m. - 7:00 p.m. On trains arriving or departing 30th Street Station, Suburban and Jefferson Station",
-        "evening_weekend": "Valid weekdays after 7:00 p.m.; all day Saturday, Sunday and major holidays. On trains arriving or departing 30th Street Station, Suburban and Jefferson Station",
-        "advance_purchase": "Tickets available for purchase at all SEPTA offices.",
-        "onboard_purchase": "Tickets available for purchase from a train conductor aboard SEPTA regional rail trains."
-    },
-    "zones": [{
-        "name": "CCP/1",
```

Comment:
I moved this file to the public folder so that my React app could access the file contents.